subgraphs/compound-forks/src/mapping.ts

```diff
@@ -1138,6 +1191,12 @@ function updateMarket(
     underlyingToken.decimals
   );
 
+  log.warning("Updated price for {}, mantissa: {}, price: ${}", [
```

Comment:
shall we remove?

Reply:
good catch - I checked for more testing logs, and they should all be gone now